doc/examples/python/m.distance.py

```diff
@@ -68,7 +68,7 @@ def main():
     # calc distance
 
     proj_type = G_begin_distance_calculations()
-    # returns 0 if projection has no metrix (ie. imagery)
+    # returns 0 if projection has no metrics (ie. imagery)
```

Comment:
I have a question here. Is it "doesn't have metrics", or "is not metric".
The source of [G_begin_distance_calculations](https://grass.osgeo.org/programming8/gis_2distance_8c.html#a988f3691493c06d4c7410d564c78c840)(void) writes the same:
![image](https://github.com/user-attachments/assets/eea00116-a542-420e-9d72-2100defa52d8)
But the 0 case would be returned by [G_database_units_to_meters_factor](https://grass.osgeo.org/programming8/defs_2gis_8h.html#a385919f1c6753d3274e44deae2f18f21)();
![image](https://github.com/user-attachments/assets/703acd22-0c50-4fa5-8e0f-930cf97da57b)

There, the documentation header is clearer. But maybe one or both of them could be wrong